moveit_ros/moveit_servo/launch/pose_tracking_example.launch.py

```diff
@@ -1,7 +1,6 @@
 import os
 from launch import LaunchDescription
-from launch.actions import ExecuteProcess
-from launch_ros.actions import Node
+import launch_ros
```

Comment:
It looks like only `Node` needs to be imported

```suggestion
from launch_ros.actions import Node
```